docs/syntax_and_semantics/assignment.md

```diff
@@ -11,45 +20,58 @@ local = 1
 
 # Assigns to a class variable
 @@class = 3
+
+# Assigns to a constant
+CONST = 4
+
+# Assigns to a setter method
+foo.method = 5
+foo[0] = 6
 ```
 
-Each of the above kinds of variables will be explained later on.
+### Method as assignment target
```

Comment:
May I suggest putting the section "Method as assignment target"  below "Combined assignments"

Reply:
Why? Setter methods are a more basic feature IMO.